examples/simulation/tree/example_calculations/calc_pressure_loss.py

```diff
@@ -0,0 +1,147 @@
+"""
+This script calculates the pressure losses and the needed power for the pump
+in a simple tree network.
+
+The assumption of three fittings has been made:
+1. Tee connector at the fork
+2. Valve at the consumer 1
+3. Valve at the consumer 2
+
+"""
+import os
+import pandas as pd
+import math
+import numpy as np
+
+path_file = os.path.dirname(__file__)
+path = os.path.abspath(os.path.join(path_file, os.pardir))
+input_data = os.path.join(path, 'input')
+result_path = os.path.join(path, 'expected_results/sequences')
+
+
+def read_data(input_value):
+    """
+    This function is reading the data of a csv with a name given as input value
+    """
+    return pd.read_csv(os.path.join(input_data, input_value + '.csv'), index_col=0)
+
+
+# Read input data for every csv component
+consumers = read_data('consumers')
+edges = read_data('edges')
+forks = read_data('forks')
+producers = read_data('producers')
+mass_flow = pd.read_csv(input_data + '/sequences/consumers-mass_flow.csv')
+
+# Constants for calculation
+rho = 971.78        # [kg/m3] TODO: later the density could be variable and calculated with CoolProp
+epsilon = 0.01      # [mm]
+zeta_tee = 2        # [-] Rough estimate of Tee connector: WJ Beek - Transport Phenomena (1999) for rougher pipes 0,75
+zeta_valve = 3.3    # [-] VDI Wärmeatlas für Nennweite von 50 mm
+mu = 0.255          # [mPa*s]
+eta_pump = 0.8      # [-] TODO: Right now made up value. What should it be in model?
+g = 9.81            # [m/s2]
+pi = math.pi
+
+# Initialize variables of type dataframe (needed for later calculations)
+v, re, lambda_simp, lambda_adv, dp_diss, dp_loc, dp_loc_tee, dp_loc_valve, dp_hyd, dp = \
+    [pd.DataFrame() for variable in range(10)]
+
+# Adjust mass flows to a dataframe containing all mass flows in correct order
+mass_flow_total = mass_flow.iloc[:, 1:]  # Get mass flows of all consumers
+mass_flow_total['0'] = mass_flow['2'] + mass_flow['3']  # Calculate sum of consumer mass flows as producer mass flow
```

Comment:
I tried to run it and get a key error here. Maybe you got different input data on your computer?

Reply:
I've just fixed the key error in my recent two commits: 
https://github.com/oemof/DHNx/pull/39/commits/b34e25cba4a6310eb1e2d08f88c165b89a0c53b1
https://github.com/oemof/DHNx/pull/39/commits/0015623c063555b8ef445dd78d31639e7d05ec80